fxgl-gameplay/src/main/kotlin/com/almasb/fxgl/minigames/triggersequence/TriggerSequenceMiniGame.kt

```diff
@@ -87,13 +89,18 @@ class TriggerSequenceView(miniGame: TriggerSequenceMiniGame = TriggerSequenceMin
             override fun onActionBegin(trigger: Trigger) {
                 if (trigger is KeyTrigger) {
                     val key = trigger.key
+                    val currentTrigger = triggerViews.children[0]
 
                     circle.opacity = 1.0
 
                     if (triggerViews.children.isNotEmpty())
                         triggerViews.children.removeAt(0)
 
-                    if (miniGame.press(key)) {
+                    // Has the correct key been pressed, and is the current view between the two lines
+                    if (miniGame.press(key) &&
+                        currentTrigger.translateX >= line1.startX &&
```

Comment:
We are potentially being too strict here from the gameplay perspective. What if the current view X is less than line1.x but the right side of the view is still between line1.x and line2.x? I think we should allow it to be accepted